<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-OTN8D0S" failures="0" tests="5" name="test.First" time="2.982" errors="3" timestamp="09 Feb 2018 09:15:07 GMT" skipped="2">
  <testcase classname="test.First" name="end" time="0.000">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerExceptionat testBase.TestBase.end(TestBase.java:93)at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)at java.base/java.lang.reflect.Method.invoke(Unknown Source)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:451)at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)at org.testng.internal.Invoker.invokeMethod(Invoker.java:529)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
    </error>
  </testcase> <!-- end -->
  <testcase classname="test.First" name="firstTest" time="0.000">
    <skipped/>
  </testcase> <!-- firstTest -->
  <testcase classname="test.First" name="secondTest" time="0.000">
    <skipped/>
  </testcase> <!-- secondTest -->
  <testcase classname="test.First" name="end" time="0.001">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerExceptionat testBase.TestBase.end(TestBase.java:93)at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)at java.base/java.lang.reflect.Method.invoke(Unknown Source)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:451)at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)at org.testng.internal.Invoker.invokeMethod(Invoker.java:529)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
    </error>
  </testcase> <!-- end -->
  <testcase classname="test.First" name="statUp" time="2.981">
    <error message="Unable to create new service: ChromeDriverService
Build info: version: &#039;3.9.1&#039;, revision: &#039;63f7b50&#039;, time: &#039;2018-02-07T22:42:28.403Z&#039;
System info: host: &#039;DESKTOP-OTN8D0S&#039;, ip: &#039;192.168.0.103&#039;, os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;9.0.4&#039;
Driver info: driver.version: unknown
Command duration or timeout: 378 milliseconds" type="org.openqa.selenium.SessionNotCreatedException">
      <![CDATA[org.openqa.selenium.SessionNotCreatedException: Unable to create new service: ChromeDriverServiceBuild info: version: '3.9.1', revision: '63f7b50', time: '2018-02-07T22:42:28.403Z'System info: host: 'DESKTOP-OTN8D0S', ip: '192.168.0.103', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '9.0.4'Driver info: driver.version: unknownCommand duration or timeout: 378 millisecondsat java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)at java.base/java.lang.reflect.Constructor.newInstance(Unknown Source)at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:214)at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:166)at org.openqa.selenium.remote.JsonWireProtocolResponse.lambda$new$0(JsonWireProtocolResponse.java:53)at org.openqa.selenium.remote.JsonWireProtocolResponse.lambda$getResponseFunction$2(JsonWireProtocolResponse.java:91)at org.openqa.selenium.remote.ProtocolHandshake.lambda$createSession$0(ProtocolHandshake.java:123)at java.base/java.util.stream.ReferencePipeline$3$1.accept(Unknown Source)at java.base/java.util.Spliterators$ArraySpliterator.tryAdvance(Unknown Source)at java.base/java.util.stream.ReferencePipeline.forEachWithCancel(Unknown Source)at java.base/java.util.stream.AbstractPipeline.copyIntoWithCancel(Unknown Source)at java.base/java.util.stream.AbstractPipeline.copyInto(Unknown Source)at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source)at java.base/java.util.stream.FindOps$FindOp.evaluateSequential(Unknown Source)at java.base/java.util.stream.AbstractPipeline.evaluate(Unknown Source)at java.base/java.util.stream.ReferencePipeline.findFirst(Unknown Source)at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:126)at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:73)at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:142)at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:601)at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:219)at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:142)at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:155)at testBase.TestBase.init(TestBase.java:79)at test.First.statUp(First.java:30)at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)at java.base/java.lang.reflect.Method.invoke(Unknown Source)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:451)at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)at org.testng.internal.Invoker.invokeMethod(Invoker.java:516)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)Caused by: org.openqa.selenium.SessionNotCreatedException: Unable to create new service: ChromeDriverServiceBuild info: version: '3.9.1', revision: '63f7b50', time: '2018-02-07T22:42:28.403Z'System info: host: 'DESKTOP-OTN8D0S', ip: '192.168.0.103', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '9.0.4'Driver info: driver.version: unknownBuild info: version: '3.8.1', revision: '6e95a6684b', time: '2017-12-01T18:33:54.468Z'System info: host: 'DESKTOP-OTN8D0S', ip: '192.168.0.103', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '9.0.4'Driver info: driver.version: unknownat org.openqa.selenium.remote.server.ServicedSession$Factory.lambda$get$0(ServicedSession.java:128)at org.openqa.selenium.remote.server.ServicedSession$Factory.apply(ServicedSession.java:145)at org.openqa.selenium.remote.server.ActiveSessionFactory.lambda$apply$12(ActiveSessionFactory.java:176)at java.util.stream.ReferencePipeline$3$1.accept(Unknown Source)at java.util.stream.ReferencePipeline$11$1.accept(Unknown Source)at java.util.stream.ReferencePipeline$2$1.accept(Unknown Source)at java.util.Spliterators$ArraySpliterator.tryAdvance(Unknown Source)at java.util.stream.ReferencePipeline.forEachWithCancel(Unknown Source)at java.util.stream.AbstractPipeline.copyIntoWithCancel(Unknown Source)at java.util.stream.AbstractPipeline.copyInto(Unknown Source)at java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source)at java.util.stream.FindOps$FindOp.evaluateSequential(Unknown Source)at java.util.stream.AbstractPipeline.evaluate(Unknown Source)at java.util.stream.ReferencePipeline.findFirst(Unknown Source)at org.openqa.selenium.remote.server.ActiveSessionFactory.apply(ActiveSessionFactory.java:179)at org.openqa.selenium.remote.server.NewSessionPipeline.lambda$null$2(NewSessionPipeline.java:65)at java.util.stream.ReferencePipeline$3$1.accept(Unknown Source)at java.util.stream.ReferencePipeline$2$1.accept(Unknown Source)at java.util.Collections$2.tryAdvance(Unknown Source)at java.util.stream.ReferencePipeline.forEachWithCancel(Unknown Source)at java.util.stream.AbstractPipeline.copyIntoWithCancel(Unknown Source)at java.util.stream.AbstractPipeline.copyInto(Unknown Source)at java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source)at java.util.stream.FindOps$FindOp.evaluateSequential(Unknown Source)at java.util.stream.AbstractPipeline.evaluate(Unknown Source)at java.util.stream.ReferencePipeline.findFirst(Unknown Source)at org.openqa.selenium.remote.server.NewSessionPipeline.lambda$createNewSession$3(NewSessionPipeline.java:68)at java.util.stream.ReferencePipeline$3$1.accept(Unknown Source)at java.util.stream.ReferencePipeline$3$1.accept(Unknown Source)at java.util.stream.ReferencePipeline$3$1.accept(Unknown Source)at java.util.stream.DistinctOps$1$2.accept(Unknown Source)at java.util.stream.ReferencePipeline$2$1.accept(Unknown Source)at java.util.stream.ReferencePipeline$3$1.accept(Unknown Source)at java.util.stream.ReferencePipeline$2$1.accept(Unknown Source)at java.util.stream.Streams$StreamBuilderImpl.tryAdvance(Unknown Source)at java.util.stream.Streams$ConcatSpliterator.tryAdvance(Unknown Source)at java.util.stream.ReferencePipeline.forEachWithCancel(Unknown Source)at java.util.stream.AbstractPipeline.copyIntoWithCancel(Unknown Source)at java.util.stream.AbstractPipeline.copyInto(Unknown Source)at java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source)at java.util.stream.FindOps$FindOp.evaluateSequential(Unknown Source)at java.util.stream.AbstractPipeline.evaluate(Unknown Source)at java.util.stream.ReferencePipeline.findFirst(Unknown Source)at org.openqa.selenium.remote.server.NewSessionPipeline.createNewSession(NewSessionPipeline.java:71)at org.openqa.selenium.remote.server.commandhandler.BeginSession.execute(BeginSession.java:65)at org.openqa.selenium.remote.server.WebDriverServlet.lambda$handle$0(WebDriverServlet.java:242)at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)at java.util.concurrent.FutureTask.run(Unknown Source)at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)at java.lang.Thread.run(Unknown Source)]]>
    </error>
  </testcase> <!-- statUp -->
</testsuite> <!-- test.First -->
